modules/lib/lib-node/src/test/resources/test/FindNodesByQueryHandlerTest.js

```diff
@@ -64,3 +64,47 @@ exports.queryEmpty = function () {
     assert.assertJsonEquals(expectedEmptyJson, result);
 };
 
+exports.sort = function () {
+    var result = repo.query({
+        'start': 0,
+        'count': 100,
+        'query': {},
+        'sort': [
+            {
+                'field': '_modifiedTime',
+                'direction': 'DESC'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8706041)
```suggestion
                "direction": 'DESC'
```